_docs_v7/Markers-and-BC.md

```diff
@@ -121,6 +123,40 @@ A marker can be defined as a Farfield boundary by addings its name to the `MARKE
 MARKER_FAR= (farfield)
 ```
 
+## Turbulence Boundary Condition ##
+
+| Solver | Version | 
+| --- | --- |
+| `RANS`, `INC_RANS`, | 7.0.0 |
```

Comment:
7.3 ?

Reply:
I think the idea was "from version X you can do Y"

Reply:
They are not updated.

Reply:
ok, clear, I thought it was supposed to be "tested for version 7.0.0"